ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -446,6 +475,25 @@ def add_br_if_to_docker(self, bridge, ext_if, int_if):
 
         VMTopology.iface_up(int_if, pid=self.pid)
 
+    def add_br_if_to_netns(self, bridge, ext_if, int_if):
```

Comment:
I think the function name is a little bit misleading. Either use a better name or add some docstring to explain what is done by this function?